### PR TITLE
Add MakeProjection command

### PR DESCRIPTION
**Pull request by MatthewBowles**
Adds MakeProjection command to CLI.

**To test:**
```
import mslice.cli as mc
ws_in = mc.load('path_to_file')
mc.MakeProjection(ws_in, '|Q|', 'DeltaE')
```

Fixes #307

### CODE REVIEW

**Comment by martyngigg** [comment]
This isn't related to this I don't think but I seem to have an issue with my load step:
```python
ws_in = mc.load('/media/data1/ExternalData/Misc/mslice/MAR21335_Ei60.00meV.nxs')
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-3-c55d48513202> in <module>()
----> 1 ws_in = mc.load('/media/data1/ExternalData/Misc/mslice/MAR21335_Ei60.00meV.nxs')

/media/data1/source/github/mantidproject/mslice/mslice/cli/_mslice_commmands.py in load(path)
     88         raise RuntimeError('could not find the path %s' % path)
     89     MAIN_WINDOW.dataloader_presenter.load_workspace([path])
---> 90     return get_workspace_handle(ospath.basename(path).split('.')[0])
     91 
     92 

/media/data1/source/github/mantidproject/mslice/mslice/models/workspacemanager/workspace_provider.pyc in get_workspace_handle(workspace_name)
     10     if isinstance(workspace_name, Workspace):
     11         return workspace_name
---> 12     return _loaded_workspaces[workspace_name]
     13 
     14 

KeyError: 'MAR21335_Ei60'
```

**Comment by MatthewBowles** [comment]
That was caused by a change in this PR. Should be fixed now.

**Comment by MatthewBowles** [comment]
@martyngigg I'm drawing a blank, any idea why python 3 might be getting a seg fault here?

**Comment by martyngigg** [comment]
@MatthewBowles It's not entirely clear. `readline.so` is an odd place - I assume that's related to IPython...

**Comment by martyngigg** [comment]
Are the other pull requests seeing the same issue?

**Comment by MatthewBowles** [comment]
@martyngigg Yes, it's the same on #317

**Comment by MatthewBowles** [comment]
@martyngigg I've patched out the ipython console and it's passing now.

File: mslice/models/projection/powder/mantid_projection_calculator.py [hunk 1]
```diff
@@ -33,15 +33,16 @@ def calculate_projection(self, input_workspace_name, axis1, axis2, units):
 
         # can have Q-E or 2theta-E or their transpose.
         if axis1 != DELTA_E_LABEL and axis2 != DELTA_E_LABEL:
-            raise NotImplementedError('Must have an energy axis')
+            raise NotImplementedError("Must have a 'DeltaE' axis")
```
Comment (martyngigg) [inline mslice/models/projection/powder/mantid_projection_calculator.py]: Can we use % DELTA_E_LABEL here too? It's all consistent if we change the label string in 1 place then.

**Pull request merged**